Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:14.04
+
+MAINTAINER WebOps MoJ <webops@digital.justice.gov.uk>
+
+# Support tools for building Python apps
+#   - build-essential
+#   - python-dev
+#   - python-setuptools
+#
+# Specific packages required for the app to work
+#   - git
+#   - libgit2-dev
+#   - libffi-dev
+#   - libyaml-dev
+# Specific packages for libgit2:
+#   - cmake
+RUN apt-get update && apt-get install -y \
+    build-essential python-dev python-setuptools \
+    git libgit2-dev libffi-dev libyaml-dev \
+    cmake
+
+# Install libgit2 - http://www.pygit2.org/install.html
+RUN mkdir /tmp/libgit2
+WORKDIR /tmp/libgit1
+ADD https://github.com/libgit2/libgit2/archive/v0.22.0.tar.gz /tmp/libgit1/
+RUN tar zxf v0.22.0.tar.gz
+# Perform the build
+WORKDIR /tmp/libgit1/libgit2-0.22.0
+RUN cmake .
+RUN make
+RUN make install
```

Comment:
We'll need libssh2 support for salt-shaker to be able to clone most/all of our repos